.github/workflows/linux.yml

```diff
@@ -49,7 +49,8 @@ jobs:
         run: |
           echo "Build command : ./scripts/build.sh --build --release"
           ./scripts/build.sh --build --release --package \
-          --target x86_64-unknown-linux-gnu,aarch64-unknown-linux-gnu
+          --target x86_64-linux-gnu,aarch64-linux-gnu \
+          x86_64-unknown-linux-gnu,aarch64-unknown-linux-gnu
```

Comment:
Do we need the unknown here still?

Reply:
I think for compat if someone has an automated script that fetches the libs